### PR TITLE
Enable ort-genai release pkg for cuda 12.2

### PR DESCRIPTION
**Pull request by gunandrose4u**
Enable ort-genai release python pkg for cuda 12.2

### CODE REVIEW

File: .pipelines/stages/jobs/py-packaging-job.yml [hunk 1]
```diff
@@ -81,6 +81,12 @@ jobs:
   - name: os
     value: ${{ parameters.os }}
 
+  - name: feed_name
+    ${{ if eq(parameters.cuda_version, '12.2') }}:
```
Comment (skyline75489) [inline .pipelines/stages/jobs/py-packaging-job.yml:85]: Better also check `eq(parameters.ep, 'cuda`)

Comment (gunandrose4u) [reply .pipelines/stages/jobs/py-packaging-job.yml:85]: why

Comment (skyline75489) [reply .pipelines/stages/jobs/py-packaging-job.yml:85]: All other packages (DirectML, ROCm) needs the original feed.

Comment (gunandrose4u) [reply .pipelines/stages/jobs/py-packaging-job.yml:85]: Make sense. In the future, when cuda default value switch to 12.2. We do not need change here.

**Review by baijumeswani — APPROVED** [review]

**Review by snnn — APPROVED** [review]

**Review by snnn — COMMENTED** [review]
I don't see a new yaml file is added. 
Please setup two new pipelines in ADO: one for nuget CUDA 12, one for python CUDA 12. And they should be triggered every day, so that we will have nightly packages published.

**Comment by baijumeswani** [comment]
@snnn until now, we only have a manual trigger to run ort-genai packaging.
But I think we should add a new yml to support nightly packaging for all the packages: ort-genai cpu, dml, cuda11.8, cuda 12.2.

But the matrix can get really big (similar to ort). We will probably need to prioritize which packages we want to run nightly.

**Comment by snnn** [comment]
If you don't have a nightly pipeline if there was any thing wrong with the setting, the problem will only be found when you are doing a release, which could make your release process very tough.

**Comment by baijumeswani** [comment]
You're right. The goal should be to have all the packages build nightly. I'll sync with the team to see if we can prioritize this work.

**Pull request merged**